hugo/assets/scss/components/tag.scss

```diff
@@ -19,7 +22,9 @@
 
     &:focus-visible,
     &:hover {
-        --tag-background: #{ $c-white };
+        --tag-background: #DDE3FD;
+        
+        transform: scale(1.03);
```

Comment:
the hover is a bit heavy now and transition is missing which is making it a 'clunky' , especially when there's going to be multiple tags.. Maybe just change the background color? 
Also put the hex variable in the colors.scss